app/controllers/application_controller.rb

```diff
@@ -1,2 +1,40 @@
 class ApplicationController < ActionController::Base
+  before_action :set_default_meta_tags
+
+  private
+
+  def set_default_meta_tags
+    title = 'Hotwire.io'
+    description = "Community-driven documentation and resource Hub for Hotwire."
+    image = "#{root_url}og-image.png"
+
+    tags = {
+      site: title,
+      reverse: true,
+      separator: '·',
+      description: description,
```

Comment:
Does it make sense to also provide `canonical` and `noindex` here?
```suggestion
      description: description,
      canonical: request.original_url,
      noindex: Rails.env.local?,
```